packages/blade/src/components/Tabs/_decisions/decisions.md

```diff
@@ -0,0 +1,385 @@
+  </TabList>
+
+  <TabPanel value="payments">
+    <Payments />
+  </TabPanel>
+  <TabPanel value="refunds">
+    <Refunds />
+  </TabPanel>
+  <TabPanel value="disputes">
+    <Disputes />
+  </TabPanel>
+  <TabPanel value="settlements">
+    <Settlements />
+  </TabPanel>
+</Tabs>
+```
+
+<img src="./example-1.png" width="380" alt="Tabs Example 1" />
+
+#### Controlled, Uncontrolled
+
+```js
+const Controlled = () => {
+  const [selected, setSelected] = React.useState('payments');
+
+  return (
+    <Tabs value={selected} onChange={(value) => setSelected(value)}>
+      <TabList>
+        <TabItem value="payments">Payments</TabItem>
+        <TabItem value="refunds">Refunds</TabItem>
+      </TabList>
+
+      <TabPanel value="payments">
+        <Payments />
+      </TabPanel>
+      <TabPanel value="refunds">
+        <Refunds />
+      </TabPanel>
+    </Tabs>
+  );
+};
+
+const Uncontrolled = () => {
+  return (
+    <Tabs defaultValue="refunds">
+      <TabList>
+        <TabItem value="payments">Payments</TabItem>
+        <TabItem value="refunds">Refunds</TabItem>
+      </TabList>
+
+      <TabPanel value="payments">
+        <Payments />
+      </TabPanel>
+      <TabPanel value="refunds">
+        <Refunds />
+      </TabPanel>
+    </Tabs>
+  );
+};
+```
+
+## Vertical Orientation
+
+```jsx
+<Tabs orientation="vertical">
+  <TabList>
+    <TabItem value="all-expenses" leading={RupeeIcon}>
+      All Expenses
+    </TabItem>
+    <TabItem
+      value="categories"
+      leading={EditIcon}
+      trailing={<Counter value={26} variant="notice" />}
+    >
+      Categorise
+    </TabItem>
+    <TabItem
+      value="review"
+      leading={TransactionIcon}
+      trailing={<Counter value={99} variant="information" />}
+    >
+      Review
+    </TabItem>
+    <TabItem value="synced" leading={CheckCircleIcon}>
+      Synced
+    </TabItem>
+    <TabItem
+      value="synced-fail"
+      leading={WarningIcon}
+      trailing={<Counter value={3} variant="negative" />}
+    >
+      Synced Failed
+    </TabItem>
+    <TabItem value="excluded" leading={SlashIcon}>
+      Excluded
+    </TabItem>
+  </TabList>
+  ...
+</Tabs>
+```
+
+<img src="./example-4.png" width="380" alt="Tabs Example 2" />
+
+## Filled Variant
+
+```jsx
+<Tabs defaultValue="refunds">
+  <TabList>
+    <TabItem value="week">Week</TabItem>
+    <TabItem value="month">Month</TabItem>
+    <TabItem value="custom">Custom</TabItem>
+  </TabList>
+
+  <TabPanel value="week">
+    <WeekGraph />
+  </TabPanel>
+  <TabPanel value="month">
+    <MonthGraph />
+  </TabPanel>
+  <TabPanel value="custom">
+    <CustomGraph />
+  </TabPanel>
+</Tabs>
+```
+
+<img src="./example-2.png" width="380" alt="Tabs Example 2" />
+
+## With Tooltip
+
+```jsx
+<Tabs>
+  <TabList>
+    <TabItem value="payments">Payments</TabItem>
+    <TabItem value="refunds">
+      <Tooltip content="Return (money) in restitution, repayment, or balancing of accounts.">
```

Comment:
the props table above says that the children will be string? but we are passing JSX here. will this work?

Reply:
Oh wait right, we have to keep this React.ReactNode then to accommodate tooltip usecases.

Reply:
Updated.